UnrealTournament/SystemLocalized/itt/XOpenGLDrv.itt

```diff
@@ -9,7 +9,7 @@ NoFindGL="Non riesco a trovare il driver OpenGL %ls"
 ; EN: MissingFunc="Missing OpenGL function %ls (%i)"
 MissingFunc="Manca la funzione OpenGL %ls (%i)"
 ; EN: ResFailed="Failed to set resolution"
-ResFailed="Errore nel settare la risoluzione"
+ResFailed="Errore nell'impostare la risoluzione"
```

Comment:
Remember to also check OpenGLDrv.itt and XOpenGLDrv.itt in Unreal and sync your changes there too.